client/src/components/TODOList/AddNewTODO.js

```diff
@@ -120,73 +138,104 @@ const AddTODOItem = () => {
         size="lg"
         open={openAddTODO}
         handler={handleOpen}
-        className="addTODODialog shadow-none"
+        className="shadow-none"
       >
-        <Card className="addTODOForm">
-          <h2>Add TODO Item</h2>
-          <form onSubmit={handleSubmit}>
+        <Card className="m-4">
+          <h2 className="flex flex-row justify-center">Add TODO Item</h2>
+          <form
+            className="flex flex-col justify-evenly h-[30rem] overflow-y-auto"
+            onSubmit={handleSubmit}
+          >
             <div className="inputField">
-              <label htmlFor="add-title">TODO:</label>
               <Input
-                id="add-title"
                 name="title"
                 value={formData.title}
-                label="title"
+                label="Title"
                 onChange={handleInputChange}
               />
             </div>
-            <div className="inputField">
-              <label htmlFor="add-dueDate">Due Date:</label>
-              <DatePicker
-                id="add-dueDate"
-                className="bg-orange-200"
-                selected={formData.dueDate}
-                onChange={handleDueDateInput}
-              />
+            <div className="flex flex-row justify-between flex-wrap">
```

Comment:
![image](https://github.com/awctw/study-dash/assets/56493743/385a75ea-1165-40fb-adab-948509f6ba80)

Add modal positioning of datepickers is spread out on wide resolutions.

Reply:
Umm... this was the best I can do so far to deal with the resolution issue you have highlighted earlier. I burned so many hours on this. I might need some help from the rest of the team to find a better resolution to the modal dialog resolution change issue